packages/core/src/model.ts

```diff
@@ -388,16 +390,6 @@ export class Model implements Storage {
     return generateId() as Ref<Doc>
   }
 
-  createDocument<M extends Doc> (_class: Ref<Class<M>>, values: Omit<M, keyof Doc>, _id?: Ref<M>): M {
-    const layout = {
-      _class,
-      _id: _id ?? this.generateId()
-    }
-    this.assign(layout, _class, (values as unknown) as AnyLayout)
-    // this.add(layout)
-    return (layout as unknown) as M
-  }
-
```

Comment:
Why remove this implementation and create new same one above? What's wrong with this?

Reply:
It was just two of them mostly the same, just choose one on top of file :-)

Reply:
Implementations are different, this one with optional `id`